src/main/resources/defaults/data/Banners.json

```diff
@@ -20,52 +20,36 @@
 		"gachaType": 301,
 		"scheduleId": 903,
 		"bannerType": "EVENT",
-		"prefabPath": "GachaShowPanel_A081",
-		"previewPrefabPath": "UI_Tab_GachaShowPanel_A081",
-		"titlePath": "UI_GACHA_SHOW_PANEL_A081_TITLE",
+		"prefabPath": "GachaShowPanel_A084",
+		"previewPrefabPath": "UI_Tab_GachaShowPanel_A084",
+		"titlePath": "UI_GACHA_SHOW_PANEL_A061_TITLE",
 		"costItemId": 223,
 		"beginTime": 0,
 		"endTime": 1924992000,
-		"sortId": 9998,
-		"rateUpItems4": [1034, 1014, 1048],
-		"rateUpItems5": [1060],
-		"fallbackItems5Pool2": [],
-		"weights5": [[1,80], [73,80], [90,10000]]
-	},
-	{
-		"gachaType": 400,
-		"scheduleId": 923,
-		"bannerType": "EVENT",
-		"prefabPath": "GachaShowPanel_A082",
-		"previewPrefabPath": "UI_Tab_GachaShowPanel_A082",
-		"titlePath": "UI_GACHA_SHOW_PANEL_A031_TITLE",
-		"costItemId": 223,
-		"beginTime": 0,
-		"endTime": 1924992000,
-		"sortId": 9998,
-		"rateUpItems4": [1034, 1014, 1048],
-		"rateUpItems5": [1026],
+		"sortId": 9999,
+		"rateUpItems4": [1065, 1036, 1055],
+		"rateUpItems5": [1057],
 		"fallbackItems5Pool2": [],
+		"weights4": [[1,510], [8,510], [10,10000]],
```

Comment:
These are the default weights btw (GachaBanner.java line 37), no need to copy them here

Reply:
> These are the default weights btw (GachaBanner.java line 37), no need to copy them here

indeed. but these can also be shown to someone who wants to modify them